tests/routes_tests/cbauthentication_tests/cb_login.test.js

```diff
@@ -0,0 +1,35 @@
+const dbConnection = require('../../../react-backend/database/dbConnection.js');
+const test = require('tape');
+const request = require('supertest');
+const app = require('../../../react-backend/app');
+
+test('Route cblogin | post | successful request', (t) => {
```

Comment:
When doing API testing I typically put the URL first here, for example:
```js
test('POST /api/cb/login | successful request', (t) => {});
```

What does "successful request" mean here in terms of the scenario we are testing? Is this a request where the authentication details are accepted? If so, I'd suggest being more specific, e.g. "authentication successful".

Reply:
Hmm... I was vaguely going along the lines of it being tests for when a user makes a successful POST request to this route, but I guess that makes less sense when I type it out as that's what supertest is mocking. Happy for suggestions on what you think it should be called.

Reply:
> "authentication successful"

☝️ is fine, with the test _only_ testing a request with a payload that results in successful authentication.

Other scenarios should be handled in separate test cases.